scan.go

```diff
@@ -10,6 +10,11 @@ import (
 	"gorm.io/gorm/schema"
 )
 
+const (
+	// DoubleUnderscore for FieldSchema.LookUpField or column Alias Delimiter
+	DoubleUnderscore = "__"
```

Comment:
don't really want to export the `DoubleUnderscore` to public, which won't be confuse to end users.